showcase/config/json-server/data.json

```diff
@@ -3,5 +3,20 @@
     { "uuid": "1", "username": "janedoe", "firstName": "Jane", "lastName": "Doe", "language": "en", "roles": ["admin"] },
     { "uuid": "2", "username": "johndoe", "firstName": "John", "lastName": "Doe", "language": "fr", "roles": ["manager"] },
     { "uuid": "3", "username": "chucknorris", "firstName": "Chuck", "lastName": "Norris", "language": "nl", "roles": ["developer"] }
-  ]
+  ],
+  "userprofile": [
+    {
+      "uuid": "4cc28d14-16d3-46e7-a05a-487533c4cc26",
+      "username": "doejohn",
+      "roles": ["AAH_EMPL", "AAH_MGMT", "AAH_ADMN", "AAH_TADM"],
```

Comment:
In the defined users above, the roles are developer, manager, admin. 
Here, you put more specific roles as AAH_EMPL or AAH_MGMT, I think we should keep only one kind of roles 😊

Reply:
True, I'll fix them